src/routes/AppProfile/AppSettings.tsx

```diff
@@ -29,6 +30,18 @@ const VersionListComp: React.FC<VersionListProps> = ({ activeProfile, selectedVe
                 .then(res => res.json())
     });
 
+    const selectVersion = async (uuid: string, index: number) => {
+        if(index != 0) {
```

Comment:
Styling + strict operator usage
```suggestion
        if (index !== 0) {
```